editor-components/feature/feature12/feature12.tsx

```diff
@@ -88,22 +173,77 @@ class Feature12 extends BaseFeature {
   }
 
   render() {
+    const upperTitle = this.getPropValue("upperTitle");
+    const title = this.getPropValue("title");
+    const titleBackingText = this.getPropValue("titleBackingText");
+    const itemCount = this.getPropValue("itemCount");
+    const messageCards = this.castToObject<IMessages[]>("messageCard");
+    const firstItem = this.getPropValue("message");
+    const firstCardTitle = firstItem[0].value;
+    const buttonText = firstItem[1].value;
+    const backgroundImage = firstItem[2].value;
```

Comment:
don't access values like this. please create a type for firstItem object.
[see here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-approach-props)